extensions/amp-list/1.0/amp-list.css

```diff
@@ -0,0 +1,3 @@
+amp-list {
```

Comment:
Is there are no styles for this, maybe we should remove this file?

Reply:
Removed.